fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -115,6 +116,14 @@ FixedLagSmoother::FixedLagSmoother(
     &FixedLagSmoother::startServiceCallback,
     this);
 
+  status_publisher_ = node_handle_.advertise<std_msgs::Bool>(
+    ros::names::resolve(params_.status_topic),
+    1,
+    true);
+  auto status = std_msgs::Bool();
+  status.data = false;
+  status_publisher_.publish(status);
```

Comment:
Maybe move this to a function?
If we at any point store the state in a member, might will be easier to maintain.

```
inline void notifyStatus(const bool running)
{
  // Update status topic
  auto status = std_msgs::Bool();
  status.data = running;
  status_publisher_.publish(status);
}
```